src/main/java/com/hubspot/jinjava/objects/SafeString.java

```diff
@@ -0,0 +1,19 @@
+package com.hubspot.jinjava.objects;
+
+public class SafeString {
+
+  private final String value;
+
+  public SafeString(String value) {
+    this.value = value;
+  }
+
+  public String getValue() {
```

Comment:
is this method needed?

Reply:
@jkollmann?

Reply:
Yes its needed to resolve the token to a string when rendering

Reply:
Never-mind. Its actually not. I got mixed up with tokens.